tests/integ/test_data_source_api.py

```diff
@@ -89,6 +94,7 @@
 
 SQL_SERVER_TABLE_NAME = "AllDataTypesTable"
 ORACLEDB_TABLE_NAME = "ALL_TYPES_TABLE"
+ORACLEDB_TEST_EXTERNAL_ACCESS_INTEGRATION = "snowpark_dbapi_oracledb_test_integration"
```

Comment:
do you create on all 3 test deployment?

Reply:
yes, it is on all 3 deployment now, I have tested it